easybuild/easyconfigs/s/STEAK/STEAK-master-foss-2019b.eb

```diff
@@ -0,0 +1,40 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+# You will need to download the tarball manually and put it in the folder
+# where this file is. 
+
+easyblock = 'MakeCp'
+
+name = 'STEAK'
+version = 'master'
+
+homepage = 'https://omictools.com/steak-tool'
+description = """Detects integrations of any sort in high-throughput 
+sequencing (HTS) data. STEAK was built for validating and discovering 
+transposable element (TE) and retroviral integrations in a variety of 
+HTS data. The software performs on both single-end (SE) and paired-end (
+PE) libraries and on a variety of HTS sequencing strategies. It can 
+be applied to a broad range of research interests and clinical uses 
+such as population genetic studies and detecting polymorphic integrations."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+sources = ['steak.zip']
+checksums = ['c5bd7d5aeeedd67e5fc17c043024552d05690080681c6e9be154b007b36f1f5a']
+
+dependencies = [
+    ('Boost', '1.71.0'),
+    ('Python', '2.7.16'),
+]
+
+modextrapaths = {'PATH': ''}
```

Comment:
This doesn't seem right, or necessary. The binary was placed under bin, which should be automatically be added to path

Reply:
When I tried it out it was not working for me which is why I left it in there. Must have been a glitch, it worked now (or the --rebuild command did not do what I expected it to do).